venues/ICLR.cc/2019/Conference/webfield/areachairWebfield.js

```diff
@@ -74,21 +74,29 @@ var buildNoteMap = function(noteNumbers) {
 // Ajax functions
 var loadData = function(result) {
   var noteNumbers = getPaperNumbersfromGroups(result.groups);
-  var noteNumbersStr = noteNumbers.join(',');
+  var blindedNotesP;
+  var metaReviewsP;
 
-  var blindedNotesP = Webfield.get('/notes', {
-    invitation: BLIND_SUBMISSION_ID, number: noteNumbersStr, noDetails: true
-  })
-  .then(function(result) {
-    return result.notes;
-  });
+  if (noteNumbers.length) {
```

Comment:
As I mentioned in my other review I think this would be better as `if (!noteNumbers.length) { return $.Deferred().resolve([]); }`

Reply:
It should be ready now

Reply:
looks like you are still doing "return last", is that ok? seems ok to me since (I think) it's just a style thing, so I'm going to merge it

Reply:
Yeah, I chose a bad example to comment on because in this case the code isn't actually returning, it's setting variables so both the if and the else are needed.